### PR TITLE
In release checklist, be explicit about when to push/pull

### PR DESCRIPTION
**Pull request by hadley**
It's easy to forget these, so we might as will be explicit.

cc @DavisVaughan

### CODE REVIEW

File: tests/testthat/_snaps/release.md [hunk 1]
```diff
@@ -30,6 +32,7 @@
       Wait for CRAN...
       
       * [ ] Accepted :tada:
+      * [ ] `git push`
       * [ ] `usethis::use_github_release()`
       * [ ] `usethis::use_dev_version()`
       * [ ] `usethis::use_news_md()`
```
Comment (DavisVaughan) [inline tests/testthat/_snaps/release.md:38]: Is this the right placement? Or after `use_news_md()`? Just struck me as odd

Comment (jennybc) [reply tests/testthat/_snaps/release.md:38]: Yeah I think it makes sense.

This `usethis::use_news_md()` item only applies to a very specific situation: right after your first CRAN release, if you didn't have any NEWS before.

Comment (DavisVaughan) [reply tests/testthat/_snaps/release.md:38]: `use_dev_version()` requires a push though, right?

Comment (jennybc) [reply tests/testthat/_snaps/release.md:38]: I think the actual point is: do we want *another* `git push` at the end? I.e. after all the bullet points that are basically starting a new dev version.

Comment (hadley) [reply tests/testthat/_snaps/release.md:38]: I just came here to say that too 😄

**Pull request merged**